### ISSUE TITLE
Is this project full committed?

### ISSUE THREAD

**Issue by mmSir** [tnquang1416/frame_interpolation_GAN#1]
It seems that this project misses something...
Q1: gans.py line 109 loadFirstPatchesWithFullImg() is not declared? This is used for test...
Q2: gans.py line 143 dataloader is without definition?
Q3: Where to download the dataset? I only found the data loaded from certain path in somebody's PC... Should i process the dateset myself?
Please help...

Looking forward to ur reply! Thanks!

**Comment by tnquang1416** [comment]
Thank you for pointing out the issue.

Q1 & Q2: I have corrected the name of variable and method.
Q3: In this repo, the project loads dataset from a folder which include video frames (as the bellow). As mentioned the published paper, you can download the UCF101 dataset. Please note that, you have to extract frames from the downloaded videos before using. 

![Sample of dataset folder](https://user-images.githubusercontent.com/10691469/106491012-60459180-64f1-11eb-9763-e15beb41a357.png)

**Comment by mmSir** [comment]
Thank you for your prompt reply！

There are some more questions.
Q1: In gans.py line 149 training_observer is not defined either. I guess this is used for output something while training.So I temporarily comment it out.

Q2: Question 2 is about the dataset. As mentioned in the paper, the author used a part of Body-Motion Only videos for both training and testing. And the training dataset includes 377 videos. I downloaded the whole UCF101 dataset and found it has many folders such as "BabyCrawling", "BodyWeightSquats". There is no classified label named "Body-Motion Only". So i selected 380 Body-Motion Only videos(in my opinion) myself. Is randomly picking 380 videos to train reasonable? I'm worried about the differences of dataset will lead to totally different training result. 

Q3: Is two stage networks (the interpolation frame generator network and the refinement network) is trained respectively ? Have u retrained the pix2pix model(the refinement network) or u just use their trained model ? If u trained those two networks respectively, what's ur training dataset for the refinement network?

I really appreciate your help！

**Comment by tnquang1416** [comment]
Hi,

My answers are the followings:

- Q1: It is a redundant line and has been removed. Thank you for notifying me.
- Q2: The paper trains the networks on some particular groups of sequence, and it includes 377 videos. You can find the detail in this [file](https://github.com/tnquang1416/frame_interpolation_GAN/blob/master/training_sequence.txt).
- Q3: For a fair comparison, the gan-based and the pix2pix are trained on a same dataset (as mentioned in Q2). Specifically, the pix2pix is trained from scratch.

Please feel free to leave a comment or open new issue if there is any problem.

**Comment by tnquang1416** [comment]
Closed due to inactive.
